src/components/ContactList/ContactList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import './ContactList.css';
+import Contact from '../Contact';
+
+function ContactList() {
+  const contacts = [
```

Comment:
this should be an immutable Map in constants. And imported. Not within a component.